code/modules/modular_armor/attachables.dm

```diff
@@ -137,3 +137,28 @@
 	parent.gas_transfer_coefficient -= siemens_coefficient_mod
 	parent.slowdown -= slowdown
 	return ..()
+
+/obj/item/helmet_module/attachable/mimir_environment_protection
+	name = "Mimir Environmental Helmet System"
```

Comment:
Add `\improper ` before the `Mimir Environmental Helmet System`.

Reply:
its not like this in any of the other ones